cluster/gke.go

```diff
@@ -321,6 +322,10 @@ func (c *GKECluster) CreateCluster() error {
 		// TODO status code !?
 		return errors.New(be.Message)
 	}
+	if gkeCluster.Status == statusError {
+		be := getBanzaiErrorFromError(err)
+		return errors.New(be.Message)
```

Comment:
I'd include also the status message provided by `gkeCluster` or if we don't want to display that to the user than log the status message.

Reply:
this line will always be equivalent to `return errors.New("")`, won't it?

Reply:
@orymate , you are right I will fix it